Sources/History/HistoryCoordinator.swift

```diff
@@ -168,19 +168,18 @@ final public class HistoryCoordinator: HistoryCoordinating {
         }
     }
 
-    public func burnDomains(_ baseDomains: Set<String>, tld: TLD, completion: @escaping () -> Void) {
+    public func burnDomains(_ baseDomains: Set<String>, tld: TLD, completion: @escaping (Set<URL>) -> Void) {
```

Comment:
Please add a unit tests to verify the parameter passed in the completion handler